build-common.xml

```diff
@@ -356,6 +356,9 @@
         <copy file="${test.src.dir}/log4j-test.properties" tofile="${test.classes.dir}/log4j.properties" failonerror="false"/>
         <junit fork="yes" printsummary="on" showoutput="true" failureproperty="junit.failure" tempdir="${single.test.dir}">
             <classpath refid="test.class.path"/>
+            <classpath>
+                <pathelement path="../store-conf/conf/msgs"/>
+            </classpath>
             <assertions><enable/></assertions>
```

Comment:
May be you can define the folder as a variable, like we are doing for other directory paths.
And add this directory in test.class.path